### DISCUSSION TITLE
Load schema error in documentdb

### DISCUSSION THREAD

**Discussion by SaiDevOps27** [learndevopsonline/learndevopsonline#1864] [Q&A]
hi @r-devops , @chaithanya1812 my workstation ip is 3.82.191.182...i am getting error in docdb in catalogue server
![Screenshot 2024-03-17 153122](https://github.com/learndevopsonline/learndevopsonline/assets/154890970/e3c6c609-b717-413a-8a02-395f42871f01)

**Comment by chaithanya1812** [comment]
hi @SaiDevOps27 , will you join lab session one more time
